tinymce-basics/customizing-the-table-toolbar.html

```diff
@@ -0,0 +1,583 @@
+<!--
+    Create an alterate table context toolbar! TinyMCE 5.4 introduced
+    a couple of new table APIs we can use to construct an alternative
+    table context toolbar. This demo showcases a table header
+    row/column toggle, a menu for choosing cell background color and
+    keyboard shortcuts for adding table rows and columns among other
+    things
+-->
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>An alternate table context toolbar</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5-dev/tinymce.min.js" referrerpolicy="origin"></script>
+    <script>
+        // An iconpack holds a custom button SVG icon.
+        // Tip! A custom icon pack does not have to be a separate file
+        tinymce.IconManager.add('customIcons', {
```

Comment:
Instead of this being an entirely separate icon pack you need to define for a single icon, would it be better if this just used the `editor.ui.registry.addIcon` API instead?

Reply:
I actually have that on line 512 as an alternative mean of registering an icon. I'll add some more documentation to reflect this.